.github/workflows/pv.yml

```diff
@@ -0,0 +1,94 @@
+        description: 'The Kokkos Core version to build'
+        default: ''
+        required: true
+        type: string
+
+jobs:
+  PR_PV_ONEAPI202410_SYCL:
+    name: PR_PV_ONEAPI202410_SYCL
+    runs-on: [inteloneapi-basekit-2024.2.1-0-devel-ubuntu22.04-latest-pv]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ inputs.kokkos_version }}
+          path: kokkos
+
+      - name: set_oneapi_device_selector_var
+        run: echo "export ONEAPI_DEVICE_SELECTOR=level_zero:gpu" >> $GITHUB_ENV
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cd kokkos/build
+
+          cmake \
+            -DCMAKE_BUILD_TYPE=Release \
+            -DCMAKE_CXX_COMPILER=icpx \
+            -DCMAKE_CXX_FLAGS="-fp-model=precise" \
+            -DCMAKE_INSTALL_PREFIX="$PWD/../install" \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DKokkos_ENABLE_SERIAL=ON \
+            -DKokkos_ENABLE_SYCL=ON \
+            -DKokkos_ARCH_INTEL_PVC=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_ONEDPL=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        run: |
+          cmake --build "$PWD/kokkos/build" -j 16
+          cmake --install "$PWD/kokkos/build"
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+
+          cmake \
+          -S "$PWD/.." \
+          -B "$PWD" \
+          -D BUILD_SHARED_LIBS=ON \
+          -D CMAKE_CXX_COMPILER=icpx \
+          -D CMAKE_EXPORT_COMPILE_COMMANDS:BOOL=OFF \
+          -D CMAKE_CXX_FLAGS="-fp-model=precise -Wno-pass-failed -Wno-unused-command-line-argument" \
+          -D CMAKE_INSTALL_PREFIX="$PWD/install" \
+          -D Kokkos_ROOT="$PWD/../../kokkos/install" \
+          -D KokkosKernels_ENABLE_TESTS_AND_PERFSUITE=OFF \
+          -D KokkosKernels_ENABLE_TESTS=ON \
+          -D KokkosKernels_ENABLE_PERFTESTS=ON \
+          -D KokkosKernels_ENABLE_EXAMPLES=ON \
+          -D KokkosKernels_INST_DOUBLE=ON \
+          -D KokkosKernels_INST_COMPLEX_DOUBLE=ON \
+          -D KokkosKernels_INST_ORDINAL_INT64_T=ON \
+          -D KokkosKernels_INST_ORDINAL_INT=ON \
+          -D KokkosKernels_INST_OFFSET_SIZE_T=ON \
+          -D KokkosKernels_INST_OFFSET_INT=ON
+
+      - name: build_kokkos_kernels
+        run: cmake --build "$PWD/kokkos-kernels/build" -j 16
```

Comment:
I'd bump this up to 64 or something.

Reply:
Done in [e9c6fee](https://github.com/kokkos/kokkos-kernels/pull/2676/commits/e9c6fee2fc6e943de4eee64ebb0182ebcefc6462)